spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
```

Comment:
I added the download group logic. However I don't understand how that'll give file access if the GROUP is set to nobody?